### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by ziin** [aclap-dev/video-downloadhelper#1181] [License & Purchase requests]
### Your question

Hello, I would like to request a refund.
I purchased the tool yesterday, but unfortunately, it wasn't able to help me download videos from a course with limited access. I believe the website has protection against this.
The tool works very well on other sites.

How can I receive a refund?

### Addon & CoApp version and details

version: 9.1.0.43
target: google
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"4f62cd7d61db493e"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 4,
    "dable_attempts_last": 1724357537319
}

**Comment by azizloj** [comment]
yeah same problem here :-(

**Comment by Shenyo8** [comment]
I'm having the same issue, it stopped working completely on one website that I use

**Comment by jcfrog** [comment]
@ziin Hi, you have been refunded. Should be available in a few business days.

**Reply by ziin** [reply]
@jcfrog It's been three weeks and I still haven't received the refund.

**Reply by jcfrog** [reply]
I'm so sorry, it seems the order has not been sent. My apologies. This time it's done, promised.
<img width="375" alt="Capture d’écran 2024-09-13 à 17 21 26" src="https://github.com/user-attachments/assets/378841d1-1ae4-4dcc-9014-00e2e58c12b2">

**Comment by jcfrog** [comment]
@azizloj @Shenyo8 please send an email at licenses@downloadhelper.net
